file_paths.md

```diff
@@ -69,6 +69,7 @@ The files annotated by an '*' indicate that they are created by package installa
         hiera@ *                          -> /opt/puppetlabs/puppet/bin/hiera
         mco@ *                            -> /opt/puppetlabs/puppet/bin/mco
         puppet@ *                         -> /opt/puppetlabs/puppet/bin/puppet
+        pxp-agent@ *                      -> /opt/puppetlabs/puppet/bin/pxp-agent
```

Comment:
Since, like `mcollectived`, this is a service which isn't intended to be directly executed by the user, I suggest that it not get this symlink up to the "public" `bin` directory.